lib/PHPUnitTagProvider.coffee

```diff
@@ -0,0 +1,69 @@
+DocblockTagProvider = require "./DocblockTagProvider"
+
+module.exports =
+
+##*
+# Provides autocompletion for PHP-unit annotation tags.
+##
+class PHPUnitTagProvider extends DocblockTagProvider
+    ###*
+     * @inheritdoc
+    ###
+    addSuggestions: (tagList, prefix) ->
+        suggestions = []
+
+        for tag in tagList
+            # TODO: This URL should go to the config file
+            documentationUrl = 'https://phpunit.de/manual/current/en/appendixes.annotations.html'
+
+            if tag.documentationName
+                documentationUrl += '#appendixes.annotations.' + tag.documentationName
+
+            # NOTE: The description must not be empty for the 'More' button to show up.
+            suggestions.push
+                type                : 'tag',
+                description         : tag.description
+                descriptionMoreURL  : documentationUrl
+                snippet             : tag.snippet
+                displayText         : tag.name
+                replacementPrefix   : prefix
+
+        return suggestions
+
+    ###*
+     * Retrieves a list of known docblock tags.
+     *
+     * @return {array}
+    ###
+    fetchTagList: () ->
+        return [
+            {name: '@after',                          documentationName: 'after',                          snippet: '@after',                                       description: 'The method should be called after each test method in a test case class'}
+            {name: '@afterClass',                     documentationName: 'afterClass',                     snippet: '@afterClass',                                  description: 'The static method should be called after all test methods in a test class have been run to clean up shared fixtures'}
+            {name: '@backupGlobals',                  documentationName: 'backupGlobals',                  snippet: '@backupGlobals ${1:disabled}',                 description: 'completely enable or disable the backup and restore operations for global variables'}
```

Comment:
Minor nitpick: could you make the description start with an upper case letter (consistent with the other descriptions)?